experimental/packages/otlp-exporter-base/src/platform/browser/OTLPExporterBrowserBase.ts

```diff
@@ -30,7 +30,9 @@ export abstract class OTLPExporterBrowserBase<
   ServiceRequest
 > extends OTLPExporterBase<OTLPExporterConfigBase, ExportItem, ServiceRequest> {
   protected _headers: Record<string, string>;
+  private _getHeaders: () => Record<string, string> = () => ({});
```

Comment:
If `_getHeaders` is an optional field, `_hasDynamicHeaders` can be removed, right?

```suggestion
  private _getHeaders?: () => Record<string, string>;
```

Reply:
Thx - yes, I'll refactor the conditional branch below so that  `_hasDynamicHeaders` is needed.